src/main.zig

```diff
@@ -402,6 +420,27 @@ fn isValidIdentifierName(str: []const u8) bool { // TODO: Remove after #480
 	}
 	return true;
 }
+var startup_finished: bool = false;
+var controller_mappings_window_shown: bool = false;
```

Comment:
Better use an enum with `showController`, `showNamePrompt`, `finished`.
Then you can also increment by one each time, making it easier to change order or add new startup menus.